### PR TITLE
Add and list recipes

### PR DESCRIPTION
**Pull request by ulises2607**
- Add recipe list views. It shows a recipe list with names, a delete button, and a description.
- Add delete functionality
- Redirect to the details page when the user clicks on the recipe name.

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = Recipe.all
```
Comment (SFarida) [inline app/controllers/recipes_controller.rb]: Hi @ulises2607 I think here you should list the recipe of the logged-in user as this is one of the project requirements.
You can use something like
`@recipes = current_user.recipe`
After setting the relation in the user model

**Review by SFarida — CHANGES_REQUESTED** [review]
Great job on this PR, I was able to see the list of recipes, you just need to filter the recipe of the current user.

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (ulises2607) [reply app/controllers/recipes_controller.rb]: Thank you for the suggestion partner!! 

https://github.com/ulises2607/rails-recipe-app/assets/97530670/0fe12fc8-383d-4fef-8ad8-6b749cd01eec

**Review by SFarida — APPROVED** [review]
You made it :clap:!
:heavy_check_mark: Its working perfectly, I could only see the recipes of the logged-in user
:heavy_check_mark: Linters are passing
Nothing more to add

**Pull request merged**